pkg/scheduler/eventhandlers.go

```diff
@@ -527,6 +532,26 @@ func addAllEventHandlers(
 	return nil
 }
 
+// podSchedulingPropertiesChange interprets the update of a pod and returns corresponding UpdatePodXYZ event(s).
+func podSchedulingPropertiesChange(newPod *v1.Pod, oldPod *v1.Pod) []framework.ClusterEvent {
```

Comment:
Could we return a single event and `|` the action types?

Reply:
I have talked with @sanposhiho before, this will impact the `queue_incoming_pods_total` metrics since we don't have a `PodLableAndRequestChanged`(label name of the event) event.

Reply:
I think those events aren't that important. Maybe we can use the more generic PodUpdated

Reply:
Have no idea whether will this harm users. Maybe we need feedbacks.

Reply:
Is such label value removal accepted in general? (If Yes, I'd rather follow Aldo's way.)
I agree those labels aren't very important, but at the same time, as @kerthcet said, it'd be a breaking change for those who use those labels.